07LessonSeven/in-class-project/css/my-style.css

```diff
@@ -0,0 +1,62 @@
+body {
+  width: 850px;
+}
+#sec1 {
+  padding-top: 225px;
+  text-align:center;
+  height: 225px;
+  background-image: url(../img/nasa-background.jpg);
+}
+h1 {
+  color:white;
+  font-size: 4em;
+  font-family: sans-serif;
+  margin-top: 0px;
+}
+#sec2 {
+  background-color: #f0f0f0;
+  height: 324px;
+  margin-top: 20px;
+  font-family: sans-serif;
+}
+#sec2 h2 {
+  font-size: 1.2em;
+}
+#sec2 p {
+  font-size: .8em;
+}
+#div1 {
+  float: left;
+  margin:15px;
+  width: 250px;
+}
```

Comment:
notice that all three of these divs have the same styling. this could be a class instead or even a child selector. think lazier!